### DISCUSSION TITLE
Does paketo-buildpacks/syft support bindings ?

### DISCUSSION THREAD

**Discussion by ettingshausen** [paketo-buildpacks/feedback#57] [Q&A] [answered]
[Syft](https://github.com/paketo-buildpacks/syft) buildpack depend on anchore-syft, does it support bindings `dependency-mapping` like [bellsoft-liberica](https://github.com/paketo-buildpacks/bellsoft-liberica) ?

**Comment by dmikusa** [comment, marked as answer]
Yes, all Paketo Buildpacks support dependency mappings.

**Reply by ettingshausen** [reply]
Thanks for your reply.
I tried [bellsoft-liberica](https://github.com/paketo-buildpacks/bellsoft-liberica) by add binding  
 `${user.home}/develop/docker/bindings/bellsoft-jdk-config:/platform/bindings/bellsoft-jdk-config`   
it worked  just fine.  
>
>bellsoft-jdk-config
├── c1afaea35fae3c921851c301bf68795c9adbb24575eb64f7194cdae48afb3f56
└── type

But I can't get syft to work with binding path `/platform/bindings/syft` or `platform/bindings/syft-config`

What is the correct path ？

**Reply by dmikusa** [reply]
The path shouldn't really matter, other than that your bindings are present under `/platform/bindings/<folder-with-bindings>/` (you don't need separate folders per buildpack either. I usually have just one big folder with all my dependency mappings. It is the `type` file that is how dependency mappings are found.

PS. you might also try https://github.com/dmikusa/binding-tool. It is a tool I wrote which can automate creating the binding files. The README has usage instructions.

**Reply by ettingshausen** [reply]
Get!🚀
